test/filter-test.js

```diff
@@ -0,0 +1,171 @@
+import { expect } from 'chai';
+import { filterRecipes } from '../src/recipes.js'
+import { sampleRecipeData } from '../src/data/sample-recipes.js';
+
+describe ('filter', function() {
+  
+  it('should return an array of filtered recipes by a tag', function() {
+    let filteredRecipes = filterRecipes(sampleRecipeData, 'starter')
```

Comment:
We should default to `const` and not `let` to keep in line with industry standards, we're not reassigning this value so `const` should work

Reply:
smart